guard/src/commands/mod.rs

```diff
@@ -16,18 +26,15 @@ mod tracker;
 //
 // Application metadata
 pub const APP_NAME: &str = "cfn-guard";
-pub const APP_VERSION: &str = env!("CARGO_PKG_VERSION");
-// Commands
-pub const PARSE_TREE: &str = "parse-tree";
-pub const RULEGEN: &str = "rulegen";
-pub const TEST: &str = "test";
-pub const VALIDATE: &str = "validate";
-pub const COMPLETIONS: &str = "completions";
+
 // Arguments for validate
 pub const ALPHABETICAL: (&str, char) = ("alphabetical", 'a');
+#[allow(dead_code)]
```

Comment:
Just curious - why are we suppressing these instead of deleting the unused declarations or commenting them out? Are they consumed elsewhere or needed later and I'm just missing it?

Reply:
These are actually used, the reason why we suppress them is cause theyre used for integration tests. Integration tests dont respect the `#[cfg(test)]` attribute either, so we need to suppress the lints